batch/scripts/create_vector_schema.sh

```diff
@@ -29,7 +29,8 @@ ogr2ogr -f "PostgreSQL" PG:"password=$PGPASSWORD host=$PGHOST port=$PGPORT dbnam
      "$LOCAL_FILE" "$SRC_LAYER" \
      -nlt PROMOTE_TO_MULTI -nln "$DATASET.$VERSION" \
      -lco GEOMETRY_NAME="$GEOMETRY_NAME" -lco SPATIAL_INDEX=NONE -lco FID="$FID_NAME" \
-     -t_srs EPSG:4326 -limit 0
+     -oo GEOM_POSSIBLE_NAMES="$GEOMETRY_NAME" -oo KEEP_GEOM_COLUMNS=NO \
+     -t_srs EPSG:4326 -s_srs EPSG:4326 -limit 0
```

Comment:
This makes sense, but please leave a comment on why you set `s_srs` since it is not right away clear for an outsider to understand the logic.